config/runtime.exs

```diff
@@ -41,8 +44,10 @@ validator_file = Keyword.get(args, :validator_file)
 enable_beacon_api = Keyword.get(args, :beacon_api, false)
 beacon_api_port = Keyword.get(args, :beacon_api_port, 4000)
 
-config :lambda_ethereum_consensus, LambdaEthereumConsensus.ForkChoice,
-  checkpoint_sync_url: checkpoint_sync_url
+if not is_nil(testnet_dir) and not is_nil(checkpoint_sync_url) do
+  IO.puts("Both checkpoint sync and testnet url specified (only one should be specified).")
```

Comment:
should we do this validation in another place so that we have access to Logger?

Reply:
Hmmm in general, other validations (like the one for mode) in this file that need to make the system crash are of the same style. Where could we move it?

Reply:
I think using `IO.puts` for these input errors is the best way to handle them. We can add the ERROR prefix manually or even the whole log format by invoking the formatter directly.